.circleci/config.yml

```diff
@@ -477,6 +481,35 @@ jobs:
           name: generate repeat_writer_info
           command: cf run-task crt-portal-django  -c "python crt_portal/manage.py generate_repeat_writer_info" --name generate-repeat-writer-info
 
+commands:
+  acquire-lock:
+    description: "Make sure nothing else is deploying."
+    parameters:
+      app:
+        type: enum
+        enum: ["clamav-rest", "crt-portal-django"]
+    steps:
+      - run:
+          name: "Wait for other deployments to finish"
+          no_output_timeout: 2h
```

Comment:
What made you pick 2h timeout and 5 minute sleep? Just curious where the numbers came from.

Reply:
The five minute sleep was just for tests (that was long enough that I could start up two other branches and push empty commits to them)

The 2h timeout means that if a deployment is queued for more than two hours, it will fail. At our current team size 2 hours should be plenty even if we merged all of our work for the week at the same time.